### PR TITLE
Rotate Symfony log file

### PR DESCRIPTION
**Pull request by jf-viguier**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Activate Symfony log file rotation. Log file can be huge, it's better for the planet to rotate this logs, especially dev logs. Log conservation scheme set by PS_LOG_MAX_FILES that can be overrided.
| Type?             | improvement
| Category?         | CO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | N/A
| How to test?      | Generate a notice or warning et check the generated log file.
| Possible impacts? | N/A
| Sponsor company   | Creabilis.com

### CODE REVIEW

**Comment by kpodemski** [comment]
I understand the idea, but it should probably be configurable, at least with the constant.

cc @PrestaShop/prestashop-maintainers

**Comment by jf-viguier** [comment]
Log file name will change : dev.log will become dev-2022-10-12.log
I guess it's not a BC break ?

**Comment by jf-viguier** [comment]
@kpodemski where can I put this 2 constants ?

**Review by FabienPapet — DISMISSED** [review]

**Comment by FabienPapet** [comment]
@kpodemski Not sure about this configuration. I think editing configuration files is enough.

**Review by lartist — DISMISSED** [review]

**Comment by lartist** [comment]
> @kpodemski Not sure about this configuration. I think editing configuration files is enough.

I agree. I don't see any advantage doing this.

**Review by lartist — DISMISSED** [review]

**Review by FabienPapet — DISMISSED** [review]

**Comment by kpodemski** [comment]
If you change these files, they'll be replaced the next time you upgrade the software unless you think of some override file that I'm unaware of.

**Comment by jf-viguier** [comment]
@kpodemski I have added a PS_LOG_MAX_FILES env param that can be overrided. Thanks to review with @FabienPapet

**Review by kpodemski — DISMISSED** [review]

File: app/config/config.yml [hunk 1]
```diff
@@ -15,6 +15,7 @@ parameters:
   prestashop_views: "%kernel.root_dir%/../src/PrestaShopBundle/Resources/views"
   admin_page: "%prestashop_views%/Admin"
   env(PS_LOG_OUTPUT): "%kernel.logs_dir%/%kernel.environment%.log"
+  env(PS_LOG_MAX_FILES): 10
```
Comment (kpodemski) [inline app/config/config.yml]:
```suggestion
  env(PS_LOG_MAX_FILES): 30
```

Comment (eternoendless) [inline app/config/config.yml]: I think 30 might be a better default

Comment (jf-viguier) [reply app/config/config.yml]: Changed to 30

Comment (jf-viguier) [reply app/config/config.yml]: Changed to 30

**Review by kpodemski — APPROVED** [review]

**Review by PululuK — APPROVED** [review]

**Review by FabienPapet — APPROVED** [review]

**Comment by Progi1984** [comment]
@jf-viguier It's a QA :heavy_check_mark: 

Steps to tests : 
* Install PrestaShop with the PR
* Go to the backoffice
* :heavy_check_mark: Check that `var/logs/dev-2022-10-31.log` is created
* Copy more than 30 tiles the file `dev-2022-10-31.log` decreasing the date (`dev-2022-10-30.log`, `dev-2022-10-29.log`, ...)
* Remove the file `dev-2022-10-31.log`
* Go to the backoffice
* :heavy_check_mark: Check that `var/logs/dev-2022-10-31.log` is created
* :heavy_check_mark: Check that only `var/logs/dev-2022-10-*.log` files are prensent in logs directory

**Comment by jolelievre** [comment]
Thanks @jf-viguier and @Progi1984

**Pull request merged**